Il2CppInspector.GUI/MainWindow.xaml.cs

```diff
@@ -390,7 +399,9 @@ public partial class MainWindow : Window
                     areaBusyIndicator.Visibility = Visibility.Visible;
 
                     await Task.Run(() => {
-                        var idaWriter = new IDAPythonScript(model);
+                        var idaWriter = new IDAPythonScript(model) {
+                            UnityVersion = ((UnityHeader)cboUnityVersion.SelectedItem)?.MinVersion,
```

Comment:
This causes an unhandled exception because `cboUnityVersion` can only be accessed from the UI thread. Read `SelectedItem` into a temporary variable before the `await` line.

Reply:
I don't know why this didn't crash for me - maybe some new VS trickery? Definitely a bug that I'll fix.

Reply:
It won't crash 100% of the time. Accessing a UI element from a non-owning thread leads to "undefined behaviour" in WPF.